bin/.constants.sh

```diff
@@ -6,12 +6,30 @@ umask 0022
 scriptsDir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
 featureDir="$scriptsDir/../features"
 self="$(basename "$0")"
+build_os="$(uname -s)"
 
-options="$(getopt -n "$BASH_SOURCE" -o '+' --long 'flags:,flags-short:,help:,usage:,sample:' -- "$@")"
+# Determinate the OS for GNU tools
+if [ "Darwin" == $build_os ]; then
+
+    # Test for possible pathes of getopt
+    # in different brew locations
+    if [[ -f "/usr/local/opt/gnu-getopt/bin/getopt" ]]; then
+        getopt_gnu="/usr/local/opt/gnu-getopt/bin/getopt"
+    fi
+
+    if [[ -f "/opt/homebrew/opt/gnu-getopt/bin/getopt" ]]; then
+        getopt_gnu="/opt/homebrew/opt/gnu-getopt/bin/getopt"
+    fi
```

Comment:
`$(brew --prefix)/opt/gnu-getopt/bin/getopt` is a bit more flexible

Reply:
We should take care that there's also the possibility to use `MacPorts` instead of `(Home)Brew` which may also deliver packages. I think we should keep this and also add `/opt/local/bin/` (MacPorts) path. What do you think?

Reply:
Fair point. The MacPorts path should be definitely added here as well.

Reply:
Good point, definitely! I've adjusted the `README.md` (see also: https://github.com/gardenlinux/gardenlinux/blob/bf44b9ed7f04a0fc609faefdbdbce0fdfc613357/README.md#build-options). Is this fine for you?